ui/src/components/TestCases/SepBlock.tsx

```diff
@@ -76,6 +73,10 @@ export const SepBlock: React.FC<{
       ProgressIcon = <FailedIcon />;
     }
 
+    if (runState === RunState.running) {
+      ProgressIcon = <LoadingIcon />;
+    }
```

Comment:
this will at least, for now, keep the progress icon from changing until the tests finish running.

Will swap in a new icon from Charles once we have it